<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>networkClass</class>
 <widget class="QMainWindow" name="networkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>network tool</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget{border:1px solid #808080;background-color:#333333;color:#DBDBDB;}
QWidget:disabled{color:#909090;}
QLineEdit{border:0px;padding:0px 3px 0px 3px;background-color:#191919;}
QCheckBox,QLabel{border:0px;}
QCheckBox::indicator{width:18px;height:18px;}
QCheckBox::indicator:unchecked{border-image:url(resources/checkbox.png) 0 0 0 0 repeat repeat;}
QCheckBox::indicator:checked{border-image:url(resources/checkbox.png) 72 0 0 0 repeat repeat;}
QPushButton:hover{background-color:#636363;}
QComboBox QAbstractItemView {outline: 0px;}
QComboBox QAbstractItemView::item:selected {background-color: #636363;}
QProgressBar{border:1px solid #808080;background-color:#191919;}
QScrollBar{border:0px;background-color:#292929;width:14px;}
QScrollBar::handle{border:1px solid #636363;background-color:#333333;margin: 16px 0px 16px 0px;min-height:30px;}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: transparent;
}</string>
   </property>
   <widget class="QProgressBar" name="progressBar_u">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>600</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>419</y>
      <width>670</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QWidget{background-color:black;color:white;}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_url">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>链接地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>请求参数：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_arg">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_d">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>600</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>上传进度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>下载进度：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_abort">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>411</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>请求方式</string>
    </property>
    <widget class="QCheckBox" name="cb_download">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>下载</string>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_upload">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>上传</string>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_get">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>GET</string>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_post">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>POST</string>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文件保存位置：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_saveDir">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_uploadFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>待上传文件：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_browser1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_browser2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_useDefault">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>默认值</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>目标文件名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_filename">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_multiDownload">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>多线程下载通道数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>注：带后缀，空则根据url获取</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>progressBar_u</zorder>
   <zorder>textEdit_output</zorder>
   <zorder>lineEdit_url</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit_arg</zorder>
   <zorder>progressBar_d</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>btn_abort</zorder>
   <zorder>btn_start</zorder>
   <zorder>label_5</zorder>
   <zorder>lineEdit_saveDir</zorder>
   <zorder>lineEdit_uploadFile</zorder>
   <zorder>label_6</zorder>
   <zorder>btn_browser1</zorder>
   <zorder>btn_browser2</zorder>
   <zorder>cb_useDefault</zorder>
   <zorder>label_8</zorder>
   <zorder>lineEdit_filename</zorder>
   <zorder>cmb_multiDownload</zorder>
   <zorder>label_11</zorder>
   <zorder>label_14</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar{border:0px;background-color:#333333;color:#DBDBDB;padding:1px 5px 1px 5px;}
QMenuBar:item{border:0px;background-color:transparent;color:#DBDBDB;}
QMenuBar:item:selected{background-color:#636363;color:#DBDBDB;}
</string>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="styleSheet">
     <string notr="true">QMenu {background-color:#333333;border:1px solid #333333;color:#DBDBDB;outline:0px;}
QMenu::item {background-color:#636363;color:#DBDBDB;border:0px;height:22px;padding:1px 5px 1px 5px;}
QMenu::item:selected {background-color:#333333;}</string>
    </property>
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About Network Tool</string>
   </property>
  </action>
  <action name="actionUsage">
   <property name="text">
    <string>Usage</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
